cmd/fluxd/main.go

```diff
@@ -232,17 +253,57 @@ func main() {
 		os.Exit(0)
 	}
 
+	viper.SetConfigName(config.ConfigName)
+	viper.AddConfigPath(config.ConfigPath)
+
+	// If there's no config file, fine. If there IS a config file, but it's garbage, we need to exit(>0).
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+			fmt.Fprintf(os.Stderr, "Info: config file not found: %s\n", err.Error())
+		} else {
+			fmt.Fprintf(os.Stderr, "Error: found config file at %s but failed to load it: %s\n", viper.ConfigFileUsed(), err.Error())
+			os.Exit(2)
+		}
+	} else {
+		fmt.Fprintf(os.Stderr, "Error: using configuration at %s, with command-line overrides\n", viper.ConfigFileUsed())
+	}
+	// Bind Viper to the pflags defined above
+	viper.BindPFlags(fs)
+
+	var config config.Config
+	viper.Unmarshal(&config)
```

Comment:
Is the returned error intentionally ignored?

Reply:
Nop